test/command/suite/select/function/string/string_slice/nth/options/default_value/with_length.expected

```diff
@@ -0,0 +1,38 @@
+plugin_register functions/string
```

Comment:
Could you use `with_length`/`without_length` or `has_length`/`no_length` in all test file names?
(I prefer to `with_length`/`without_length`.)